recipe/meta.yaml

```diff
@@ -8,8 +8,6 @@ build:
   number: 1001
 
 requirements:
-  host:
-    - python
```

Comment:
This is a meta package and thus we don't need to expand a full `python` build matrix. Once build per platform is sufficient.